community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -158,8 +160,9 @@ public void doImport( Input input ) throws IOException
             }
 
             // Stage 3 -- relationships, properties
-            final RelationshipStage relationshipStage = new RelationshipStage( relationships, idMapper,
-                    neoStore, nodeRelationshipLink, input.specificRelationshipIds() );
+            final RelationshipStage relationshipStage = new RelationshipStage(
+                    nodes.supportsMultiplePasses() ? relationships : inputCache.relationships(),
```

Comment:
Should it be `relationships. supportsMultiplePasses()` ?
Looks like `InputIterable`s for nodes and relationships might have different value of `supportsMultiplePasses`, am I right?